README.md

```diff
@@ -0,0 +1,42 @@
+A library that explores dependencies from a given module or namespace in a local Python environment, then classifies these dependencies.
+
+The library was isolated from the [LZY project](https://github.com/lambdazy/lzy)'s codebase to ensure reusability and reduce the difficulty of dependency management.
+
+If you are interested in using our library, please contact us via Github Issues.
+
+## Brief example
+
+```python
+In [1]: from envzy import AutoExplorer
+
+In [2]: explorer = AutoExplorer(pypi_index_url='https://pypi.org/simple', additional_pypi_packages={}, target_python=(3, 9))
+
+In [3]: explorer.get_local_module_paths({'foo': AutoExplorer})
```

Comment:
argument meaning `{'foo': AutoExplorer}` could be unclear for users reading README
may be better:
```py
namespace = {'foo': AutoExplorer}
explorer.get_local_module_paths(namespace)
```